src/jobs/remindersJob.ts

```diff
@@ -0,0 +1,95 @@
+import ERC20 from '../models/ERC20';
+import { ERC721 } from '../models/ERC721';
+import MailService from '../services/MailService';
+import AccountProxy from '../proxies/AccountProxy';
+import ejs from 'ejs';
+import path from 'path';
+import { DASHBOARD_URL, API_URL } from '../config/secrets';
+import { AssetPool } from '../models/AssetPool';
+import { Withdrawal } from '../models/Withdrawal';
+import { logger } from '../util/logger';
+
+export const sendRemindersJob = async () => {
+    const subject = 'THX Reminder';
+    let title: string, message: string;
+    const accounts = await AccountProxy.getActiveAccountsEmail();
+
+    // REMINDERS BASED ON THE ACCOUNT
```

Comment:
@valeriagrazzini Hey! Please use normal casing for comments or remove your notes when publishing the PR :)